Header header
geometry_msgs/Point[] centroids
geometry_msgs/Vector3[] sizes
int8[] types
